fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomeManager.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.EDGE_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.HILLS_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.INJECTED_BIOME_LIST;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.RIVER_MAP;
+import static net.fabricmc.fabric.impl.biomes.BiomeLists.SHORE_MAP;
+
+import net.fabricmc.fabric.impl.biomes.BiomeAssociate;
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.util.Pair;
+import net.minecraft.util.registry.Registry;
+import net.minecraft.world.biome.Biome;
+
+public class OverworldBiomeManager
+{
+	/**
+	 * Adds the biome to the specified climate group
+	 * 
+	 * @param biome
+	 * @param climate
+	 */
+	public static void addBiomeToClimate(Biome biome, BiomeClimate climate)
+	{
+		INJECTED_BIOME_LIST.add(new Pair<Integer, BiomeClimate>(Registry.BIOME.getRawId(biome), climate));
```

Comment:
This should use an adequate map like the others. As it is, it does not prevent accidentally adding the same biome multiple times to a climate, and does not offer a good way to set a weight either.

Reply:
Also you should use the diamond operator on your Pair instantiation.

Reply:
When I have time I will add a better system for weights.. I'm sure someone will come up with a better solution than I after this is merged though

Reply:
I'll can't think of an adequate weights solution where I use only one set or map, so I'll use the current list plus a map of sets, for now

Reply:
You can consider using a guava multiset for an item-to-weight mapping.

Reply:
@Pyrofab , @liach Actually, we don't want to restrict accidentally adding the same biome twice

I think it would be better for modders to just have a way to check if the biome has already been added or something. Some mods may want to inject a biome again.

(don't ask me why, I just don't think Fabric API should try restrict people)

Reply:
My solution is to, instead of restricting users, provide a function to get the weight of a biome in the specified BiomeClimate